site/pages/home/client-testimonials-slice/navigator/arrow/index.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames/bind';
+import styles from './style.css';
+
+const cx = classnames.bind(styles);
+
+type ArrowProps = {
+  direction: string,
+  onClick: Function,
+  currentIndex: number,
+};
+
+const Arrow = ({ direction, onClick, currentIndex }: ArrowProps) => {
+  const LAST_INDEX = 4;
+  const click = () => {
+    let newIndex = currentIndex;
+    if (direction === 'left' && currentIndex > 0) {
+      newIndex = currentIndex - 1;
+    } else if (direction === 'right' && currentIndex < LAST_INDEX) {
+      newIndex = currentIndex + 1;
+    }
+    onClick(newIndex);
+  };
+
+  const isClickable =
+    (direction === 'left' && currentIndex > 0) ||
+    (direction === 'right' && currentIndex < LAST_INDEX);
+
+  return (
+    <button
+      className={cx('arrow', direction === 'left' ? 'arrowLeft' : 'arrowRight')}
+      onClick={click}
+      aria-label={direction}
+    >
+      <div className={isClickable ? styles.active : styles.inactive} />
+      <div className={styles.innerArrow} />
+    </button>
+  );
+};
+
+export default Arrow;
```

Comment:
Why was it necessary to copy this component over, was it possible to share it with the one from the other slice?

Reply:
LAST_INDEX was different for the two places, but I've now made it a parameter that can be passed to the component so we can reuse the original definition